packages/table/src/Table.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useReducer, ReactElement } from 'react';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { css } from '@leafygreen-ui/emotion';
+import HeaderRow, { HeaderRowProps } from './HeaderRow';
+import TableHeader, { TableHeaderProps } from './TableHeader';
+import CheckboxCell from './CheckboxCell';
+import { State, Types, TableProvider, reducer } from './table-context';
+
+// * Clean TS
+// * Add nodeRef to Row
+
+const tableStyles = css`
+  border-collapse: collapse;
+  box-sizing: border-box;
+`;
+
+interface TableProps extends React.ComponentPropsWithoutRef<'table'> {
+  data?: Array<any>;
+  columns?:
```

Comment:
I think columns and data is required, no? Otherwise what do we render?

Reply:
👍